src/mmw/js/src/analyze/views.js

```diff
@@ -57,6 +57,33 @@ var ResultsView = Marionette.LayoutView.extend({
             newProjectUrl = '/project/new/' + modelPackageName,
             projectUrl = '/project';
 
+        if (modelPackageName === 'gwlfe') {
```

Comment:
Looks like TR-55 fails on very small AOIs too. Trying to model this one:

<img width="1254" alt="screen shot 2016-09-28 at 3 41 23 pm" src="https://cloud.githubusercontent.com/assets/4165523/18929326/0e9dc8f4-8592-11e6-8960-9bf52233e435.png">

led to this error:

```sh
[2016-09-28 15:40:44,123: ERROR/Worker-1] Bad input data to TR55: u'runoff-vol'
```